client/src/pages/portal-wide.js

```diff
@@ -1,32 +1,113 @@
 import React from 'react'
-import { Box, Text } from 'grommet'
-import { useResponsive } from 'hooks/useResponsive'
+import { Anchor, Box, Text } from 'grommet'
 import { api } from 'api'
-import Error from 'pages/_error'
+import { DatasetPortalWideDownloadCard } from 'components/DatasetPortalWideDownloadCard'
 
-// NOTE: This page is used to display CCDL datasets
-const Datasets = ({ datasets }) => {
-  const { responsive } = useResponsive()
-
-  // we don't want to 404 here we want to show that the api is down
-  if (!datasets) return <Error />
+const PortalWideDownloads = () => {
+  const metadataDataset = {
+    format: null,
+    modality: null,
+    includes_merged: null,
+    metadata_only: true,
+    size_in_bytes: 0
+  }
+  const dataDatasets = [
+    {
+      format: 'SINGLE_CELL_EXPERIMENT',
+      modality: 'SINGLE_CELL',
+      includes_merged: false,
+      metadata_only: false,
+      has_single_cell: true,
+      has_cite_seq_data: true,
+      has_bulk_rna_seq: true,
+      size_in_bytes: 429496729600 // 400 GB
+    },
+    {
+      format: 'ANN_DATA',
+      modality: 'SINGLE_CELL',
+      includes_merged: false,
+      metadata_only: false,
+      has_single_cell: true,
+      has_cite_seq_data: true,
+      has_bulk_rna_seq: true,
+      size_in_bytes: 966367641600 // 900 GB
+    },
+    {
+      format: 'SINGLE_CELL_EXPERIMENT',
+      modality: 'SPATIAL',
+      includes_merged: false,
+      metadata_only: false,
+      has_spatial_data: true,
+      has_bulk_rna_seq: true,
+      size_in_bytes: 429496729600 // 400 GB
+    }
+  ]
 
   return (
-    <Box width="full" pad={responsive({ horizontal: 'medium' })}>
-      <Box pad={{ bottom: 'large' }}>
-        <Text serif size="xlarge">
-          CCDL Datasets
-        </Text>
+    <>
+      <Box alignSelf="start" pad={{ left: 'medium' }} margin={{ top: 'none' }}>
+        <Box pad={{ bottom: 'medium' }}>
+          <Text size="xlarge" weight="bold">
+            Portal-wide Downloads
+          </Text>
+        </Box>
+        <Box pad={{ top: 'large' }}>
+          <Text size="medium" margin={{ top: 'small' }}>
+            Data from the projects in the ScPCA portal is packaged together for
+            your convenience.
+            <Anchor
+              label=" Please learn more about the portal-wide downloads here."
+              href="#"
+            />
+          </Text>
+        </Box>
+
+        <Box pad={{ top: 'large', bottom: 'xlarge' }}>
+          <Text size="large" margin={{ bottom: 'medium' }}>
+            Metadata Downloads
+          </Text>
+          <Box>
+            <DatasetPortalWideDownloadCard dataset={metadataDataset} />
+          </Box>
+        </Box>
       </Box>
-    </Box>
+
+      <Box background="#EDF7FD">
```

Comment:
You can use available colors from `client/src/theme/global.js`.

```suggestion
      <Box background="dawn">
```